Add Changelog about updates to Node-RED container tags (#3070)
FlowFuse/website: src/changelog/2025/03/container-tags.md

---
title: Changes to tags for flowfuse/node-red
description: Updating what Node-RED version the latest tag points to

> date: 2025-03-26 14:00:00.0

Author: ```suggestion
date: 2025-04-09 12:00:00.0
```

authors: ['ben-hardill']
tags:

>  - chagelog

Reviewer: ```suggestion
 - changelog
```

---

> As part of the upcoming FlowFuse 2.16.0 release we will be updating which

Reviewer: ```suggestion
As part of the upcoming FlowFuse 2.16 release due on April 10th, we will be updating which
```

> As part of the upcoming FlowFuse 2.16.0 release, due on April 10th, we will be updating which

Author: ```suggestion
As part of the upcoming FlowFuse 2.16.0 release, we will be updating which
```

> As part of the upcoming FlowFuse 2.16.0 release we will be updating which
> version the `latest` tag points to for the `flowfuse/node-red` containers.
> 
> This tag currently points to the latest version of the FlowFuse Stack container but contains Node-RED version 3.0.2 and NodeJS 16. After the update it will contain Node-RED version 4.0.9 and NodeJS 20.
> 
> It will track the latest Node-RED version as they are released.
> 
> This change will impact self hosted Docker and Kubernetes that have not changed the default stack.
> 
> You can update the tag by creating a new version of the Stack under the Admin Settings -> Stacks page as a FlowFuse Admin user.
> 
> We make the following tags available which track the latest version of FlowFuse.

Reviewer: ```suggestion
As part of the upcoming FlowFuse 2.16.0 release, due on April 10th, we will be updating which
version the `latest` tag points to for the `flowfuse/node-red` containers.

This change will impact self-hosted Docker and Kubernetes installs that have not changed the default stack.

The `latest` tag currently points to the latest version of the FlowFuse Stack container but contains Node-RED version 3.0.2 and NodeJS 16. After the update it will contain Node-RED version 4.0.9 and NodeJS 20.

It will then continue to track the latest stable Node-RED version as they are released.

You can update the tag by creating a new version of the Stack under the Admin Settings -> Stacks page as a FlowFuse Admin user.

We make the following tags available which track the latest version of FlowFuse:
```

Reviewer: Slight rewording - putting the 'who does this impact' earlier.
